cmd/dfget/app/version.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	"fmt"
+
+	"github.com/dragonflyoss/Dragonfly/version"
+
+	"github.com/spf13/cobra"
+)
+
+var versionCmd = &cobra.Command{
```

Comment:
Maybe provide our own help command? WDYT?

Reply:
I cannot understand your mean. Is that info about version is unnecessary?

Reply:
It is ok at present, but we can also do some custom output like [this](https://github.com/dragonflyoss/Dragonfly/blob/master/dfget/cli/cli_parameters.go#L106) to make it more friendly, WDYT?